### PR TITLE
:bug: :lipstick: Fix modal height on {N}

### PR DESCRIPTION
**Pull request by jakobe**

### CODE REVIEW

File: src/kirby/components/modal/modal-wrapper/modal-wrapper.component.tns.ts [hunk 1]
```diff
@@ -66,4 +88,53 @@ export class ModalWrapperComponent extends ContentView {
...
+  getStatusBarHeight() {
+    var result = 0;
+    if (application.android) {
+      var resourceId = application.android.context
+        .getResources()
+        .getIdentifier('status_bar_height', 'dimen', 'android');
+      if (resourceId) {
+        const screenScale = screen.mainScreen.scale;
+        const pixelValue = application.android.context.getResources().getDimension(resourceId);
+        result = pixelValue / screenScale;
```
Comment (jonasjensen-brolstark) [inline src/kirby/components/modal/modal-wrapper/modal-wrapper.component.tns.ts]: And you are sure, that `screenScale` never can be 0 ?

File: src/kirby/components/modal/modal-wrapper/modal-wrapper.component.tns.ts [hunk 2]
```diff
@@ -66,4 +88,53 @@ export class ModalWrapperComponent extends ContentView {
...
+      if (realHeight - displayHeight > 0) {
+        // Portrait
+        navBarHeight = realHeight - displayHeight;
+      } else if (realWidth - displayWidth > 0) {
+        // Landscape
+        navBarHeight = realWidth - displayWidth;
+      }
+
+      // Convert to device independent pixels and return
+      return navBarHeight / application.android.context.getResources().getDisplayMetrics().density;
```
Comment (jonasjensen-brolstark) [inline src/kirby/components/modal/modal-wrapper/modal-wrapper.component.tns.ts]: `application.android.context.getResources().getDisplayMetrics().density` can never be 0 ?

File: src/kirby/components/modal/modal-wrapper/modal-wrapper.component.tns.ts (hunk 1 above)
Comment (jakobe) [reply src/kirby/components/modal/modal-wrapper/modal-wrapper.component.tns.ts]: Pretty sure, it's what we already use for cards

File: src/kirby/components/modal/modal-wrapper/modal-wrapper.component.tns.ts (hunk 2 above)
Comment (jakobe) [reply src/kirby/components/modal/modal-wrapper/modal-wrapper.component.tns.ts]: Not after sdk 17 or something...

**Pull request merged**